lib/src/widgets/yust_file_picker.dart

```diff
@@ -107,16 +109,18 @@ class YustFilePickerState extends State<YustFilePicker> {
       future: _fileHandler.updateFiles(widget.files),
       builder: (context, snapshot) {
         if (kIsWeb &&
-            widget.enableDropzone && _enabled &&
+            widget.enableDropzone &&
+            _enabled &&
             (widget.allowedExtensions?.isNotEmpty ?? true)) {
           return _buildDropzone(context);
         } else {
           return YustListTile(
             suffixChild: Wrap(children: [
               if (widget.allowedExtensions != null) _buildInfoIcon(context),
               // ignore: deprecated_member_use_from_same_package
-              if (widget.allowMultiple ||
-                  (widget.numberOfFiles ?? 2) > 1 ||
+              if (widget.allowMultiple && widget.numberOfFiles != 1 ||
```

Comment:
ändere die Bedingung gerne in `widget.numberOfFiles == null || (widget.numberOfFiles != null && (widget.files.length < widget.numberOfFiles! || widget.numberOfFiles == 1 && widget.overwriteSingleFile))` und überprüf nochmal, ob das funktioniert

Reply:
Funktioniert